packages/wdio-reporter/src/stats/suite.ts

```diff
@@ -0,0 +1,48 @@
+import HookStats from './hook'
+import RunnableStats, { Runnable } from './runnable'
+import TestStats from './test'
+
+export interface Suite extends Runnable {
```

Comment:
maybe it makes sense to add an example payload from https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-reporter#events in front of all these interfaces.